src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -95,8 +95,8 @@ class DatagridCell extends React.Component {
   }
 
   getStyle() {
-    const { width } = this.props;
-    return { width, minWidth: '2rem', maxWidth: width };
+    const { width, editable } = this.props;
+    return editable ? { width, maxWidth: width } : { width, minWidth: '2rem', maxWidth: width };
```

Comment:
can you move this minWidth in SCSS like you've done for editable ? then no need for ternary

Reply:
ok, i'll fix it